oaknational/Oak-Web-Application
# Feat/lesq 362/lesson transcripts [LESQ-362]

## Description

- add gcloud storage client for fetching files from buckets
- add a method to wrap the gcloud client 
- add webvtt-parser library to parse text retrieved from the bucket into a format we can use more easily (add a type declaration file also)
- update lessonOverview schema
- create some helper methods for further parsing the output of the captions file - this is a bit unpleasant
- fetch and set transcript sentences for new content

## How to test

1. Go to a new lesson, eg. https://deploy-preview-1980--oak-web-application.netlify.thenational.academy/teachers/programmes/geography-primary-ks1/units/local-area-why-is-our-place-special/lessons/locating-our-place-with-an-address-and-directions
2. Click on show transcript under the video
3. You should see the transcript
